README.md

```diff
@@ -20,7 +20,7 @@ Installation
 
 #### pnpm
 ```bash
-pnpm add --dev @ember/test-helpers
+pnpm add -D @ember/test-helpers
```

Comment:
In documentation, i think we should always prefer long arguments so peaple have a better chance of seeing what is going on

Reply:
`````
pnpm add --dev @ember/test-helpers
 ERROR  Unknown option: 'dev'
For help, run: pnpm help add

pnpm --version
9.9.0
```

But(!) I guess we want this option (from `pnpm add --help`):

```
  -D, --save-dev                      Save package to your `devDependencies`
```

Reply:
Updated to the expanded version.